ERC20_LIST.json

```diff
@@ -648,6 +679,25 @@
         "useInSwap": false,
         "hasPermit": false,
         "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-assets/main/0_tokens/ANGLE-token/cross-chain/ANGLE-avax.svg"
+      },
+      "0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7": {
+        "address": "0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7",
+        "name": "Wrapped AVAX",
+        "decimals": 18,
+        "symbol": "AVAX",
```

Comment:
no underlying then?

Reply:
it should be 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE

Reply:
I added it under the entry 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE